### ISSUE TITLE
Default packages don't install correctly

### ISSUE THREAD

**Issue by whitfin** [asdf-vm/asdf-nodejs#67]
This appears to occur when you don't already have node.js installed (i.e. fresh system). All installs will fail as `node` is not found in the $PATH when the packages are installed.

`env: node: No such file or directory`, for reference.

**Comment by trinode** [comment]
![image](https://user-images.githubusercontent.com/965926/35760687-c97d8a2c-0879-11e8-9364-8f6a113756d7.png)
Same here

**Comment by b0o** [comment]
Same issue here. 

I was able to fix the issue by overriding the PATH environment variable when executing `npm install` in the `install_default_npm_packages()` function inside the `install` script:

```diff
install_default_npm_packages() {
  local default_npm_packages="${HOME}/.default-npm-packages"

  if [ ! -f $default_npm_packages ]; then return; fi

   for name in $(cat $default_npm_packages); do
     echo -ne "\nInstalling \e[33m${name}\e[39m npm package... "

-    if npm install -g $name > /dev/null 2>&1; then
+    PATH="$PATH:$ASDF_INSTALL_PATH/bin" npm install -g $name
+    if [[ $? -eq 0 ]]; then
       echo -e "\e[32mSUCCESS\e[39m"
     else
       echo -e "\e[31mFAIL\e[39m"

    fi
  done
}
```

**Comment by augustobmoura** [comment]
I think this was fixed a long time ago, in PRs #67 and #162. If anybody still bumps on this issue, please open a new one up. Sorry for the delay in closing this one
